WireDomain/Sources/WireDomain/Repositories/User/UserRepository.swift

```diff
@@ -50,6 +50,9 @@ public protocol UserRepositoryProtocol {
 
     func pullUsers(userIDs: [WireDataModel.QualifiedID]) async throws
 
+    /// Removes user push token from defaults.
```

Comment:
```suggestion
    /// Removes user push token from storage.
```
or 
```suggestion
    /// Removes user push token from user defaults.
```

I guess we don't want to share implementation details in public interface